packages/datadog-instrumentations/src/url.js

```diff
@@ -83,6 +77,4 @@ addHook({ name: names }, function (url) {
       }
     })
   }
-
-  return url
```

Comment:
Why remove this return value?

Reply:
It was pretty much my test case for simplifying the implementation code.